### ISSUE TITLE
Tinker's Construct Item Textures Bug

### ISSUE THREAD

**Issue by xrosbeast** [sp614x/optifine#5546]
TConstruct-1.16.5-3.0.2.78 and OptiFine_1.16.5_HD_U_G7 are conflicting in some way to cause all the tool parts to look like purple and black checkered squares.

**Comment by CyaniideDrive** [comment]
Yeah, it's happening for me too, been looking for a fix for a long time now.
